src/main/java/teammates/common/util/TeammatesDateTimeFormatter.java

```diff
@@ -0,0 +1,21 @@
+package teammates.common.util;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+/**
+ * DateTimeFormatter for TEAMMATES with default Locale.US settings.
```

Comment:
Something like this may be better?
`DateTimeFormatter with TEAMMATES specific defaults (i.e. US locale).`